### ISSUE TITLE
Website: Vacancy page

### ISSUE THREAD

**Issue by GerbenBosschieter** [teamforus/general#231]
Learn more about change requests here: https://bit.ly/39CWeEE

### Requested by:
- Team members

### Change description
- Would be nice to have a vacancy page on the website

**Comment by maxvisser** [comment]
@GerbenBosschieter Do we already have a template for a new page? How easily can this be implemented?

**Comment by GerbenBosschieter** [comment]
@maxvisser not really. Not matching our current design. Otherwise it would be an IP ;)

**Comment by GerbenBosschieter** [comment]
Status: Spoke with @linda-90 about this. She made a start with the design for this. She gonna finish this and ask team to review. If accepted we will start development

**Comment by linda-90** [comment]
Figma link: https://www.figma.com/file/KD2HGR90hiR5kDBODj7DBw/%F0%9F%92%BBWebsite?node-id=563%3A0&viewport=214%2C168%2C0.11680220812559128

**Comment by maxvisser** [comment]
Hi guys; I see the new designs and like them alot; I thought about this abit more. As a vancancy is not really static content; I see there is maybe a need for CMS functionality here. When I worked on the Blog (blog.forus.io) I thought about aswell including this in Forus but decided against it as it needs to be more rapidly released as the current frontend application.

A proposal could be to not add vacancy to forus-frontend as a new pug file but rather create for example a Google Site and add it as a subdomain (werkenbij.forus.io); so its a standalone site with a CMS.

**Comment by GerbenBosschieter** [comment]
Okay sounds good, I am not really sure how google sites works. Is it possible to add our own designs to it ?

**Comment by maxvisser** [comment]
I invited you to an example intranet site I made a while back. You can create new site on https://sites.google.com/new

Not sure if we can make the experience seamless; also not sure if this is really necessary if you make it accessible on its on domain, you can even do a dark mode version of the site (as developers like it dark)

**Comment by maartenfv** [comment]
We are going to use Zoho recruit for now. This way we can make the page fast and it is easy to manage. It also has an embed version so we can combine Zoho recruit with our own website (and design as stated above).
